src/main/scala/mesosphere/marathon/api/v2/TasksResource.scala

```diff
@@ -56,6 +59,36 @@ class TasksResource @Inject() (
     "\t"
   ))
 
+  @DELETE
+  @Produces(Array(MediaType.APPLICATION_JSON))
+  @Consumes(Array(MediaType.APPLICATION_JSON))
+  @Timed
+  def killTasks(
+    @QueryParam("scale")@DefaultValue("false") scale: Boolean,
+    body: Array[Byte]): Response = {
+    val taskIds = Json.parse(body).as[Seq[String]]
```

Comment:
Why do we use play json here? Is jersey not capable of deserializing a Seq[String]?